src/scripts/main.js

```diff
@@ -1 +1,71 @@
 'use strict';
+
+let leftMousePressed = false;
+let rightMousePressed = false;
+
+const firstPromise = new Promise(function (resolve, reject) {
+  document.addEventListener('mousedown', (ev) => {
+    if (ev.button === 0) {
+      resolve('First promise was resolved');
+      leftMousePressed = true;
+    }
+  });
+
+  setTimeout(() => {
+    if (!leftMousePressed) {
+      reject('First promise was rejected');
+    }
+  }, 3000);
+});
```

Comment:
The first promise logic is correct in terms of resolving when the left mouse button is pressed. However, the rejection logic is based solely on a timeout, which may not be the best approach if the user presses the button after 3 seconds. Consider whether this behavior aligns with the task requirements.